.github/workflows/ci.yaml

```diff
@@ -17,8 +17,6 @@ jobs:
         run: |
           go get ./pinecone
       - name: Run tests
-        run: go test ./pinecone
+        run: go test -count=1 -v ./pinecone
```

Comment:
I think the default for `count` is 1 so you can probably remove explicitly setting it.

Reply:
I thinkkkk from the [documentation](https://pkg.go.dev/cmd/go#hdr-Test_packages) it's not, actually! Check this out: 
<img width="993" alt="Screenshot 2024-07-26 at 12 13 57 PM" src="https://github.com/user-attachments/assets/1ae126e8-e01c-4e17-941d-71b1685d024f">